Sources/_CShims/include/data_shims.h

```diff
@@ -0,0 +1,20 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CSHIMS_DATA_H
+#define CSHIMS_DATA_H
+
+#include "_CShimsMacros.h"
+
+INTERNAL char *_data_mktemp(char *arg);
```

Comment:
nit: I think the pattern used in some of the other shim files like `filemanager_shims` and `string_shims` would be `_datashims_mktemp`. I'm not sure if we've really decided on that pattern, but should we be as consistent as we can be?

Reply:
They are actually sort of all over the place. I'd like to do a follow up commit to address the naming and styles.